app/javascript/crayons/Modal/Modal.jsx

```diff
@@ -41,43 +41,40 @@ export const Modal = ({
   overlay,
   onClose,
 }) => {
-  const shortcuts = {
-    escape: onClose,
-  };
-
   return (
-    <div
-      data-testid="modal-container"
-      className={`crayons-modal${getAdditionalClassNames({
-        size,
-        className,
-      })}`}
-    >
+    <FocusTrap onDeactivate={onClose}>
```

Comment:
I've just found an issue with this - when the modal is closed by clicking the close icon, `onDeactivate` is also getting triggered, so for modal usages where just flip the `isOpen` state, it flips from open -> close -> open

tl;dr some modals can't be closed by the close button right now - I'll fix it!